galaxy/main/signals/handlers.py

```diff
@@ -43,15 +43,11 @@ def user_logged_in_handler(request, user, **kwargs):
     username = social.extra_data.get('login')
     sanitized_username = username.lower().replace('-', '_')
 
-    try:
-        namespace = models.ProviderNamespace.objects.get(name=username)
+    if models.ProviderNamespace.objects.filter(name=username).exists():
```

Comment:
Won't this create a duplicate provider namespace if the user changes their username or the capitalization on their username? If so, is that a problem?

Reply:
On these changes yes a new provider namespace is created, I don't think that is an issue as we have been operating that way, and this way our system has a link to what the changed github username is via the provider namespace, since our user.username will no longer match the github username.